user/api.go

```diff
@@ -367,24 +377,13 @@ func (a *Api) UpdateUser(res http.ResponseWriter, req *http.Request, vars map[st
 				}
 			}
 
-			if updatedUser.EmailVerified && updatedUser.TermsAccepted != "" {
-				if updateUserDetails.EmailVerified != nil || updateUserDetails.TermsAccepted != nil {
-					if err := a.userEventsNotifier.NotifyUserCreated(req.Context(), *updatedUser); err != nil {
-						a.logger.Println(http.StatusInternalServerError, err.Error())
-						res.WriteHeader(http.StatusInternalServerError)
-						return
-					}
-
-				} else {
-					if err := a.userEventsNotifier.NotifyUserUpdated(req.Context(), *originalUser, *updatedUser); err != nil {
-						a.logger.Println(http.StatusInternalServerError, err.Error())
-						res.WriteHeader(http.StatusInternalServerError)
-						return
-					}
-				}
-			}
 			a.logMetricForUser(updatedUser.Id, "userupdated", sessionToken, map[string]string{"server": strconv.FormatBool(tokenData.IsServer)})
 			a.sendUser(res, updatedUser, tokenData.IsServer)
+			if err := a.userEventsNotifier.NotifyUserUpdated(req.Context(), *originalUser, *updatedUser); err != nil {
```

Comment:
`a.sendUser(res, updatedUser, tokenData.IsServer)` must be placed after this block

Reply:
done